### ISSUE TITLE
ES6 function syntax issue

### ISSUE THREAD

**Issue by alecpl** [squizlabs/PHP_CodeSniffer#2784]
For following javascript code:
```
window.axios.interceptors.response.use(
    response => {
        // Do nothing
        return response
    },
    error => {
        var error_msg

        if (error.response && error.response.status == 422) {
            // cut
        }
    }
)
```
there's two issues:
1. It complains about indentation of the comment line to be 8 instead of expected 4 spaces. phpdbf can fix it this way, but obviously it's not what should happen.
2. It complains about the same regarding `var error_msg` line, but this case is different, phpdbf is not able to fix it. Also note that it does not complain about indentation of the if statement below.

Used phpcs 3.5.3. PHP 7.4. Rules: PSR1 + PSR12.

**Comment by gsherwood** [comment]
PHP_CodeSniffer does not support modern JS syntax. All JS functionality will be removed in version 4 and no bugs will be fixed in version 3. I strongly suggest switching to a dedicated JS linting tool like eslint.

The version 4 roadmap can be found here: https://github.com/squizlabs/PHP_CodeSniffer/milestone/11

The specific issue for this is here: https://github.com/squizlabs/PHP_CodeSniffer/issues/2448
